<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parentForm</class>
 <widget class="QWidget" name="parentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>791</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel{
border : 2px solid white;
font-size: 30px;
color: white;
background-color: rgb(32,32,32);
}</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lap1">
        <property name="styleSheet">
         <string notr="true">QLabel{
border : 2px solid white;
background-color: rgb(32,32,32);
color: white;
}</string>
        </property>
        <property name="text">
         <string>Lap1: 00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lap2">
        <property name="styleSheet">
         <string notr="true">QLabel{
border : 2px solid white;
background-color: rgb(32,32,32);
color: white;
}</string>
        </property>
        <property name="text">
         <string>Lap2: 00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lap3">
        <property name="styleSheet">
         <string notr="true">QLabel{
border : 2px solid white;
background-color: rgb(32,32,32);
color: white;
}</string>
        </property>
        <property name="text">
         <string>Lap3: 00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="start">
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
    border-radius: 10px;
background-color: rgb(132,132,132);
font-size: 25px;
color: rgb(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="re_set">
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
    border-radius: 10px;
background-color: rgb(132,132,132);
font-size: 25px;
color: rgb(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>Re-set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pause">
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
    border-radius: 10px;
background-color: rgb(132,132,132);
font-size: 25px;
color: rgb(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButtonSubmitComment">
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
    border-radius: 10px;
background-color: rgb(132,132,132);
font-size: 25px;
color: rgb(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>submit comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClear">
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
    border-radius: 10px;
background-color: rgb(132,132,132);
font-size: 25px;
color: rgb(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="commentBox">
      <property name="styleSheet">
       <string notr="true">QTextEdit{
color: rgb(255,255,255);
background-color: rgb(32,32,32);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
